### PR TITLE
create project Integration added

### PR DESCRIPTION
**Pull request by Akrya**
#### Description
Works technically. Not super well designed. Can be adjusted later.

#### Review Checklist
- [X] The requirements on the description have been met
- [X] The relevant documentation has been updated:
    - [X] code comments for classes / methods / complex statements
    - [X] package readme
    - [X] documentation for high level approaches or product info
- [ ] Tests:
    - [ ] new tests written
    - [ ] existing affected tests updated
    - [ ] all green and passing
- [ ] Tested by reviewer

### CODE REVIEW

File: app/src/pages/MainPage.tsx [hunk 1]
```diff
@@ -97,21 +95,38 @@ import {
...
-            fill="outline"
-            className="ion-margin-top"
-            type="submit"
-            expand="block"
-          >
-            <IonIcon icon={add} />
-            Create
-          </IonButton>
-        </form>
+<<<<<<< HEAD
```
Comment (Cvos0162) [inline app/src/pages/MainPage.tsx:98]: I still see a merge error, can you check this again?

File: app/src/pages/ProjectPage.tsx [hunk 2]
```diff
@@ -38,15 +39,7 @@ const ProjectPage: React.FC = () => {
...
-      <IonHeader>
-        <IonToolbar className="header">
-          <IonButton fill="clear" slot="start" routerLink="/" routerDirection="back">
-            <IonIcon icon={arrowBack}/>
-            </IonButton>
-          <IonTitle slot="end">User</IonTitle>
-          <IonButton onClick={onLogout} fill="clear" slot="end" routerLink="/auth" routerDirection="back">Log out</IonButton>
-        </IonToolbar>
-      </IonHeader>
+      <Header routerLink={"/"} name={"User"}/>
```
Comment (Cvos0162) [inline app/src/pages/ProjectPage.tsx:42]: This will be changed in next merge

File: app/src/services/ProjectServices.ts [hunk 3]
```diff
@@ -55,13 +79,42 @@ async function getProjectUsers(project: string, firebase: any) {
...
-        '/projects/' + project + '/users' + '?id_token=' + localStorage.getItem('user-token'), requestOptions)
+        '/projects/' + project + '/users' + '?id_token=' + localStorage.getItem('user-token') + '&page=0&page_size=20',
+        requestOptions)
         .then(handleResponse)
         .then(data => {
             return data.users
         })
  }
 
+ async function getProjectTags(project: string, firebase:any) {
```
Comment (Cvos0162) [inline app/src/services/ProjectServices.ts:90]: master has this function in services/labelServices

File: app/src/services/ProjectServices.ts [hunk 4]
```diff
@@ -88,6 +141,32 @@ async function getProjectUsers(project: string, firebase: any) {
         })
  }
 
+ async function setProjectTags(project: string, label_name: string, firebase:any) {
```
Comment (Cvos0162) [inline app/src/services/ProjectServices.ts:144]: maybe move this over to services/labelServices too

**Review by Cvos0162 — CHANGES_REQUESTED** [review]
I hope it isn't too much to handle. I guess grab someone if need help ;)

**Comment by Akrya** [comment]
> I hope it isn't too much to handle. I guess grab someone if need help ;)

ok I think i got it. Ill fix it tomorrow morning if its still bad sorry

**Review by Cvos0162 — APPROVED** [review]
All working

**Pull request merged**